code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -180,6 +180,9 @@
 	max_ammo = 8
 	multiple_sprites = 2
 
+/obj/item/ammo_box/magazine/m10mm/empty		//for maint drops
```

Comment:
Eh, I'm not sure how the current gun code works, but doesn't this make the magazine just have space for a single bullet? That'd be super wonky.